zbus/tests/e2e.rs

```diff
@@ -430,6 +463,105 @@ impl MyIfaceImpl {
     async fn alert_count(ctxt: &SignalContext<'_>, val: u32) -> zbus::Result<()>;
 }
 
+#[derive(Debug)]
+struct MyEmitsChangedSignalIfaceImpl {
+    modifiable_default: u32,
+    modifiable_true: u32,
+    modifiable_invalidates: u32,
+    modifiable_const: u32,
+    modifiable_false: u32,
+}
+
+impl MyEmitsChangedSignalIfaceImpl {
+    fn new() -> MyEmitsChangedSignalIfaceImpl {
+        MyEmitsChangedSignalIfaceImpl {
+            modifiable_default: 0,
+            modifiable_true: 0,
+            modifiable_invalidates: 0,
+            modifiable_const: 0,
+            modifiable_false: 0,
+        }
+    }
+}
+
+#[dbus_interface(interface = "org.freedesktop.MyEmitsChangedSignalIface")]
+impl MyEmitsChangedSignalIfaceImpl {
```

Comment:
Thanks for adding this. However, I don't think we need to add another interface just for this. Why not add new properties to the existing interface and proxy? You'll basically just need 4-5 new props.